packages/core/src/main/notification/NotificationService.ts

```diff
@@ -0,0 +1,59 @@
+import {CRUDEngine} from '@wireapp/store-engine/dist/commonjs/engine/index';
+import APIClient = require('@wireapp/api-client');
+import {Notification} from '@wireapp/api-client/dist/commonjs/notification/index';
+import {RecordNotFoundError} from '@wireapp/store-engine/dist/commonjs/engine/error';
+
+export default class NotificationService {
+  public static STORES = {
+    AMPLIFY: 'amplify',
```

Comment:
Looking back it was a terrible decission to name the table "`amplify`". Do you see a chance that we can change this name or do we need to keep it to be backward compatible?

Reply:
Unfortunately we need it to be backward compatible